mypy/stubdoc.py

```diff
@@ -141,8 +142,13 @@ def format_sig(
             retfield = " -> " + ret_type
 
         prefix = "async " if is_async else ""
-        sig = "{indent}{prefix}def {name}({args}){ret}:".format(
-            indent=indent, prefix=prefix, name=self.name, args=", ".join(args), ret=retfield
+        sig = "{indent}{prefix}def {name}{type_args}({args}){ret}:".format(
+            indent=indent,
+            prefix=prefix,
+            name=self.name,
+            args=", ".join(args),
+            ret=retfield,
+            type_args=self.type_args,
         )
```

Comment:
nit: while we're touching this, we could take the opportunity to switch to an f-string:
```suggestion
        sig = f"{indent}{prefix}def {self.name}{self.type_args}({', '.join(args)}){retfield}:"
```